nextflow.config

```diff
@@ -192,7 +192,7 @@ profiles {
         docker.enabled         = true
         podman.enabled         = false
         shifter.enabled        = false
-        docker.runOptions      = '-u $(id -u):$(id -g)'
+        docker.runOptions      = '-u $(id -u):$(id -g) -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro'
```

Comment:
Here we have `docker.userEmulation   = { params.use_gatk_spark ? false : true }.call()`  on the master-branch.